index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My portfolio website</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <!-- Simulate a smartphone / tablet -->
+    <div class="mobile-container">
+      <!-- Top Navigation Menu -->
+      <div class="navbar">
+        <a href="#home" class="active">Welcome</a>
+        <div class="dropdown">
+          <button class="dropbtn">
+            <i class="fa fa-bars"></i>
+          </button>
+          <div class="dropdown-content">
+            <a href="#">Portfolio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="main">
+      <div class="background">
+        <img class="header" src="image/header.png" alt="wallpaper" />
+        <h1>Hey there.I'm Blessing</h1>
+        <h2>I'm a software developer</h2>
+        <p>
+          I can help you build a product,<br />feature or website. Look through some of my work an
+          experience! If you like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <div class="icons">
```

Comment:
For this icon, let's do the same with the comment above by using `ul`/`li` instead.